jsons/Nations.json

```diff
@@ -1678,7 +1710,7 @@
 		"outerColor": [153,5,3],
 		"innerColor": [244,232,54],
 		"uniqueName": "Achaemenid Legacy",
-		"uniques": ["Golden Age length increased by [50]%","[+1] Movement <for [All] units> <during a Golden Age>","[+10]% Strength <for [All] units> <during a Golden Age>"],
+		"uniques": ["[+50]% Golden Age length increased","[+1] Movement <for [All] units> <during a Golden Age>","[+10]% Strength <for [All] units> <during a Golden Age>"],
```

Comment:
'increased' should be removed as well iirc

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/e30a5cd741368e657e331407583cd63bd024e8c2